scripts/build/windows.bat

```diff
@@ -193,13 +193,13 @@ if not "%arch%"=="%electron_arch%" (
 
 move %output_build_directory%\Etcher-win32-%arch% %package_output%
 
-call bash.exe scripts\unix\electron-create-asar.sh^
+call bash.exe scripts\build\electron-create-asar.sh^
  -d %package_output%\resources\app^
  -o %package_output%\resources\app.asar
 
 call rimraf %package_output%\resources\app
 
-call bash.exe scripts\windows\electron-sign-exe.sh^
+call bash.exe scripts\build\electron-sign-exe-win32.sh^
```

Comment:
It's a good thing that I know this script won't be hanging around for very long, otherwise I'd have asked for all these `win32` strings to be replaced with variables ;-D